Lib/zipfile.py

```diff
@@ -295,6 +295,7 @@ class ZipInfo (object):
         'filename',
         'date_time',
         'compress_type',
+        '_compress_level',
```

Comment:
I've prefixed this with an underscore because it's not exposed when reading back. `ZipInfo` has another "private" attribute, `_raw_time` [below](https://github.com/python/cpython/pull/5385/files#diff-30e3ce7dc73ea0f34b323c729c652a08R313).